src/cluster_migrateslots.c

```diff
@@ -892,6 +892,10 @@ void clusterCommandMigrateSlots(client *c) {
                                 (sds)c->argv[curr_index]->ptr);
             goto cleanup;
         }
+        if (target_node == server.cluster->myself) {
+            addReplyError(c, "Slots are served by myself.");
```

Comment:
Maybe we can return other errors, such as the target node is myself, `Slots are served by myself` seems a bit strange in this condition.

Reply:
"by myself" is not correct English grammar. It should be "by me", right? Or maybe "by this node" sounds better?

`myself` is internal code terminology.